doc/thrust.dox

```diff
@@ -2057,12 +2057,8 @@ INCLUDE_FILE_PATTERNS  =
 # recursively expanded use the := operator instead of the = operator.
 # This tag requires that the tag ENABLE_PREPROCESSING is set to YES.
 
-PREDEFINED             = THRUST_NOEXCEPT=noexcept \
-                         "THRUST_DEFAULT={}" \
-                         "THRUST_NODISCARD=[[nodiscard]]" \
+PREDEFINED             = "THRUST_NODISCARD=[[nodiscard]]" \
```

Comment:
I'm in the middle of completely redoing the doxygen setup, so it may be best to leave this alone for now.

Reply:
Understood. I'll leave these in for now, since the alternative is to have macros in the doxyfile that no longer exist in the codebase. We can resolve conflicts later.